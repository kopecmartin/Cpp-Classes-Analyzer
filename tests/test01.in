class A {};
class C {virtual int foo(void) =0;};
class B : C {};
class D : A, B {};
