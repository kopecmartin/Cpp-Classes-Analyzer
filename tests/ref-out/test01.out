<?xml version="1.0" encoding="UTF-8"?>
<class name="C" kind="abstract">
    <private>
        <methods>
            <method name="foo" type="int" scope="instance">
                <virtual pure="yes" />
                <arguments></arguments>
            </method>
        </methods>
    </private>
</class>
